lib/service_base_url_test_kit/version.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 module ServiceBaseURLTestKit
-  VERSION = '0.11.1'
+  VERSION = '0.11.1'          # TODO breaking change
```

Comment:
why is this a breaking change?

Reply:
I started putting this comment because at the time I wasn't sure who be putting up the release PR. Since inferno core is going from 0.5 -> 0.6 (which is breaking) this test kit also needs a breaking version bump 0.11 -> 0.12.